src/app/modules/pictureapp-explore/components/photo-uploader-dialog/components/photo-uploader-stepper/photo-uploader-stepper.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, OnInit, Output, EventEmitter, Input } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { FileUploader } from 'ng2-file-upload';
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+import { PhotoUploaderModel } from 'src/app/models/photo-uploader-model';
+import { environment } from 'src/environments/environment';
+
+@Component({
+  selector: 'app-photo-uploader-stepper',
+  templateUrl: './photo-uploader-stepper.component.html',
+  styleUrls: ['./photo-uploader-stepper.component.scss']
+})
+export class PhotoUploaderStepperComponent implements OnInit {
+  @Input() fileInput: File;
+
+  isLinear = false;
+  isOptional = false;
+  firstFormGroup: FormGroup;
+  secondFormGroup: FormGroup;
+  currentFileUploader: FileUploader;
+  test: string;
+  public uploader: FileUploader;
+  public filePreviewPaths: SafeUrl[];
+
+  public hasBaseDropZoneOver = false;
+  baseUrl = environment.apiUrl;
+
+  constructor(private _formBuilder: FormBuilder, private sanitizer: DomSanitizer) { }
+
+  ngOnInit() {
+    this.firstFormGroup = this._formBuilder.group({
+      firstCtrl: [''],
+      fileInput: ['', Validators.required]
+    });
+    this.secondFormGroup = this._formBuilder.group({
+      secondCtrl: ''
+    });
+
+    this.initUploader();
+    this.setCurrentPhotoForUploader();
+    this.prepareImagePreviewForDropedImages();
+  }
+
+
+  initUploader() {
+    this.uploader = new FileUploader({
```

Comment:
Why do we use different upload approaches? Here is a FileUploader whilst couple files above there is HttpClient?